src/property-filter/controller.ts

```diff
@@ -147,6 +155,14 @@ export const parseText = (
   };
 };
 
+// We need to remove the first leading space in case the user presses space
+// after the operator, for example: Owner: admin, will result in value of ` admin`
+// and we need to remove the first space, if the user added any more spaces only the
+// first one will be removed.
+function trimLeadingSpace(text: string) {
+  return text.charAt(0) === ' ' ? text.slice(1) : text;
```

Comment:
Why not `trimStart`?

Reply:
I see, it was previous implementation as well. Fine, let's not make any changes there.